packages/components/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 -   `Modal`: Modal dialog small improvement for elementShouldBeHidden ([#65941](https://github.com/WordPress/gutenberg/pull/65941)).
 -   `Tabs`: revamped vertical orientation styles ([#65387](https://github.com/WordPress/gutenberg/pull/65387)).
+-   `ComboboxControl`: display `No items found` when there are no matches found ([#66142](https://github.com/WordPress/gutenberg/pull/66142)) 
```

Comment:
Looks like this also affects the `FormTokenField` component when the `__experimentalExpandOnFocus` prop is enabled.